OBSERVATION-WEATHER-API-BLUEPRINT.apib

```diff
@@ -72,6 +72,20 @@ code  description                           comment
 The 'shipCourseText' returns one of the following values: "not moving", "NE", "E", "SE", "S", "SW", "W", "NW", "N", "unknown".
 The 'shipSpeedRangeInKnots' returns one of the following values: "0", "1-5", "6-10", "11-15", "16-20", "21-25", "26-30", "31-35", "36-40", ">40", "unknown".
 
+It is important to use encoded timestamps when doing the requests to Point Observation Service by for example POSTMAN or browser.
+To encode properly the timestamp
+1. go to https://www.urlencoder.org/ site
+2. paste desired timestamp
+3. without changing any parameters click "> ENCODE <" button
+4. use generated encoded timestamp in the url
```

Comment:
To encode properly the timestamp:
1. Go to https://www.urlencoder.org/ site.
2. Paste desired timestamp.
3. Without changing any parameters click "> ENCODE <" button.
4. Use generated encoded timestamp in the url.